bin/doc-test

```diff
@@ -7,12 +7,13 @@ program
   .version(pkg.version)
   .option('-f, --file [pathToFile]','File needs to be tested', 'README.md')
   .option('-d, --debug [true | false]','Retain test file after execution')
+  .option('--inspect','Debug the test file.', 7000)
```

Comment:
I would default to 9229, consistent with `node --inspect-brk`

Reply:
If the doc test is started with node debugger it will crash. 
`node --inspect-brk node_modules/.bin/doc-test --inspect` will crash if we default to 9229.

Reply:
are you talking about debugging doc-test itself, or users debugging their tests?

Reply:
imo the most important thing is to be consistent with the node debugging flags, even if it means that maintainers of this library end up having to explicitly specify non-colliding ports in their dev/test workflows

```sh
node --inspect-brk node_modules/.bin/doc-test --inspect 7000
```

Reply:
Sure. That makes sense I will update the PR.

Reply:
Updated PR.